tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -0,0 +1,101 @@
+using Azure.Sdk.Tools.Cli.Services;
+using System.Text;
+using CsvHelper;
+using CsvHelper.Configuration;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ILabelHelper
+    {
+        public LabelHelper.ResultType CheckServiceLabel(string csvContent, string serviceName);
+        public string CreateServiceLabel(string csvContent, string serviceLabel);
+        public string NormalizeLabel(string label);
+        public List<LabelData> getLabelsFromCsv(string csvContent);
+    }
+
+    public class LabelHelper(ILogger<LabelHelper> logger) : ILabelHelper
+    {
+        internal const string ServiceLabelColorCode = "e99695"; // color code for service labels in common-labels.csv
+
+        public List<LabelData> getLabelsFromCsv(string csvContent)
+        {
+            using var reader = new StringReader(csvContent);
+            using var csvReader = new CsvReader(reader, config);
+            return csvReader.GetRecords<LabelData>().ToList();
+        }
+
+        private static CsvConfiguration config = new CsvConfiguration(System.Globalization.CultureInfo.InvariantCulture)
+        {
+            HasHeaderRecord = false,
+            NewLine = "\n",
+        };
+
+        public enum ResultType
```

Comment:
`ResultType` is redundant because the enum itself is a type declaration. I'd use something like `ServiceLabelStatus`